common/src/main/java/com/wynntils/features/user/DummyOverlayFeature.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.google.common.collect.ImmutableList;
+import com.mojang.blaze3d.platform.Window;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.overlays.Overlay;
+import com.wynntils.core.features.overlays.OverlayPosition;
+import com.wynntils.core.features.overlays.annotations.OverlayFeature;
+import com.wynntils.core.features.overlays.annotations.OverlayInfo;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.mc.render.RenderUtils;
+import com.wynntils.utils.objects.CustomColor;
+
+@OverlayFeature
+public class DummyOverlayFeature extends UserFeature {
+
+    @SuppressWarnings("UnusedDeclaration")
```

Comment:
I don't think changing access modifiers would help, I think we can live without the SuppressWarning tough, I only added it because IDEA complained.

Reply:
Ah, it's just IDEA? I thought it was the normal javac. 

IDEA can be told to consider annotations as accesses. I know we did that for `@SubscribeEvent` methods in legacy, otherwise they seemed to be never called. Don't remember how to do it though...

Reply:
I tested around a bit. You can get rid of the warning by clicking on "Show quick fixes" and selecting "Suppress unused warning if annotated by ...". I don't know if we can set this as some kind of project default that we can distribute with the source. Possibly, but I don't know exactly how.

Reply:
It's stored in .idea/misc.xml as a `<component name="EntryPointsManager">`. I think we can create a checked-in version of this data in some way. At least my local misc.xml contains stuff like net.minecraftforge.fml.common.Mod.EventHandler that I don't remember having added.

Reply:
Sadly, the only choice seem to be to include the entire .idea/misc.xml in the git repo. :( I think this is a bad idea. But you can export an example config that can be imported into IntelliJ. I think that's the best thing we can do, or describe in a README how to get rid of those warnings yourself.